src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stop work");
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck stop work");
```

Comment:
The message in `doWork` method is incorrect. It should print a message indicating that the truck is starting work, such as `"Truck start work"`.